public/app/features/dashboard/components/Inspector/PanelInspector.tsx

```diff
@@ -390,7 +393,7 @@ const getStyles = stylesFactory(() => {
       margin-left: 16px;
     `,
     tabContent: css`
-      height: calc(100% - 32px);
+      height: 100%;
```

Comment:
this looks better to me... but maybe I am missing something

Reply:
This might be unnecessary now. This was done before Torkels scrolling fix for the table.